.docker/keycloak/docker-compose-keycloak-local.yml

```
@@ -31,17 +30,17 @@ services:
     ports:
       - "5433:5432"
     volumes:
-      - kamann_keycloak_db_volume:/var/lib/postgresql/data
+      - kamann_keycloak_db_local_volume:/var/lib/postgresql/data
     networks:
       - kamann_network
     healthcheck:
-      test: [ "CMD-SHELL", "pg_isready -U keycloak_user" ]
+      test: [ "CMD-SHELL", "pg_isready -U keycloak_user -d keycloak_db" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)